chain/store/snapshot.go

```diff
@@ -269,21 +279,21 @@ func (s *walkScheduler) work(ctx context.Context, todo *walkTask, results chan *
 	results <- &walkResult{c: todo.c}
 
 	// extract relevant dags to walk from the block
-	if todo.taskType == 0 {
+	if todo.taskType == Block {
 		blk := todo.c
 		data, err := s.store.Get(ctx, blk)
 		if err != nil {
 			return err
 		}
 		var b types.BlockHeader
 		if err := b.UnmarshalCBOR(bytes.NewBuffer(data.RawData())); err != nil {
-			return xerrors.Errorf("unmarshaling block header (cid=%s): %w", blk, err)
+			return xerrors.Errorf("unmarshalling block header (cid=%s): %w", blk, err)
 		}
-		if b.Height%1000 == 0 {
+		if b.Height%1_000 == 0 {
```

Comment:
hmmm....what does this mean?

Reply:
emit a log message every 1000 blocks
the underscore is serving as a comma for readability https://go.dev/doc/go1.13#language